src/index.html

```diff
@@ -1,14 +1,51 @@
 <!doctype html>
+
+
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header>
+      <img
+        src="./images/logo.png"
+        class="logo"
+        alt="logo"
+      >
+      <nav>
+        <ul class="nav__menu">
```

Comment:
you don't have `nav` block, so you can't use `nav__menu` element here. rename this or add class `.nav` to parent element